### DISCUSSION TITLE
Is there a way to automate window resizing with Rectangle ?

### DISCUSSION THREAD

**Discussion by julescllt** [rxhanson/Rectangle#1213] [Q&A]
Hi,

I'd like to know if there's a way for me to let Rectangle automatically resize any app that I open, maybe with Automator or something alike ? I'd like to use it to set a 'default window size' somehow, which I haven't found any way to easily do on my Mac. My windows never open the way I want them to so I have to use the keyboard shortcut everytime I use an app.

Best

**Comment by rxhanson** [comment]
Rectangle can be automated using URLs, as noted in the [readme](https://github.com/rxhanson/Rectangle#execute-an-action-by-url).

The specific functionality that you're looking for is on my todo list to be included as part Rectangle Pro, just haven't gotten to it yet.

**Reply by julescllt** [reply]
Thank you. Hadn't thought to use the URLs this way. Will make sure to buy it when you do include it!
Best
